package.json

```diff
@@ -74,6 +74,7 @@
     "classnames": "^2.1.3",
     "eslint-cli": "^1.1.1",
     "history": "^4.7.2",
+    "lodash": "^4.17.11",
```

Comment:
Finally hehe 🙈 
Fredrik was against this as we only needed a few functions and didn't want to bloat the codebase, but I think it's way more beneficiary to utilize this as we could leverage lodash in multiple places to reduce code and improve readability.